Snakefile

```diff
@@ -888,26 +880,29 @@ rule run_scenario:
     resources:
         mem_mb=5000,
     run:
-        from scripts.build_test_configs import create_test_config
-        import yaml
-
-        # get base configuration file from diff config
-        with open(input.diff_config) as f:
-            base_config_path = (
-                yaml.full_load(f)
-                .get("run", {})
-                .get("base_config", "config.tutorial.yaml")
-            )
```

Comment:
Hi @davide-f, we already had a review iteration. Not sure about this bit @davide-f . **I will check this PR over the next weeks and ping you if there is a review needed.** It would be good to get this monster merged during the next month.